using_images/s2i_images/ruby.adoc

```diff
@@ -96,6 +96,12 @@ and ruby gem installation. The application assets are only compiled if
 Asset compilation only happens when the application runs in a production
 environment. Therefore, you can use this variable when assets have already been
 compiled.
+
+|`*PUMA_MIN_THREADS*`, `*PUMA_MAX_THREADS*`
+|Indicates the minimum and maximum threads that will be available in link:https://github.com/puma/puma[Puma]'s thread pool.
```

Comment:
Are there any limits on the number that can be specified for each variable? Or is it valid as long as PUMA_MIN_THREADS < PUMA_MAX_THREADS ?